etp-backend/src/main/clj/solita/etp/api/energiatodistus.clj

```diff
@@ -63,7 +64,7 @@
                                  (schema/optional-key :offset) schema/Int
                                  (schema/optional-key :where)  schema/Str}}
             :responses  {200 {:body [energiatodistus-schema/Energiatodistus]}}
-            :access     rooli-service/laatija?
+            :access     (some-fn rooli-service/laatija? rooli-service/paakayttaja?)
```

Comment:
some-fn:ää tulee kyllä itse käytettyä harmillisen vähän, mut kaunistahan sillä saa.

Reply:
Funktion nimi on ehkä vähän huono valittu. Ramdassa tämän funktion nimi on: anyPass ja vastaavasti allPass on clojuren every-pred.

Reply:
Clojuressa nämä higher order and ja or on nimetty siis every-pred / some-fn